config/bash/cobbler.completion

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+_cobbler_completions()
+{
+    local cur prev cobbler_type
+    declare -A opts
+
+    cur="${COMP_WORDS[COMP_CWORD]}"
+    prev="${COMP_WORDS[COMP_CWORD-1]}"
+    cobbler_type=${COMP_WORDS[1]}
+    COMPREPLY=()
+    TYPE="distro profile system repo image mgmtclass package file aclsetup buildiso import list replicate report reposync sync validateks version signature get-loaders hardlink"
+    ACTION="add edit copy list remove rename report"
+    opts=(
+        [distro]="--name --kernel --initrd --kopts --kopts-post --ksmeta --arch --breed --template-files"
+        [profile]="--name --distro --kickstart --kopts --ksmeta --virt-file-size --virt-ram --virt-type --virt-cpus --virt-path --virt-bridge --server --parent"
+        [system]="--name --profile --mac --ip-address --hostname --kopts --ksmeta --kickstart --netboot-enabled/N --server --gateway --dns-name --static-routes --power-address --power-type --power-user --power-pass --power-id"
+        [repo]="--mirror --name --rpmlist --creatrepo-flags --keep-updated --priority --arch --mirror-locally --breed"
+        [image]="--name --arch --breed --comment --ctime --mtime --file --depth --image-type --network-count --os-version --owners --parent --kickstart --virt-auto-boot --virt-bridge --virt-cpus --virt-file-size --virt-disk-driver --virt-path --virt-ram --virt-type --clobber --in-place"
+        [mgmtclass]="--name --comment --packages --files"
+        [package]="--uid --depth --comment --ctime --mtime --owners --name --action --installer --version --clobber --in-place"
+        [file]="--uid --depth --comment --ctime --mtime --owners --name --is-dir --action --group --clobber --in-place"
+    )
```

Comment:
Take care of the [second suggestion](https://github.com/cobbler/cobbler/pull/2208/files/7bd2d8308c4e4066c174721ee7c3432c8d3ebd04#r351768313).

Reply:
I'm not sure, but it seems fine now, I mark this as resolved.

Reply:
yes it's fine indeed